### ISSUE TITLE
Should any acceptance of a pull request bump versions

### ISSUE THREAD

**Issue by joshuaherman** [curbmap/Open-Street-Parking-Restriction-Specification#4]
If a pull is accepted should we bump a minor version? 
What would constitute a major version bump?

**Comment by eselkin** [comment]
I don't think we should change minor versions until we actually hit a micro version :-)... I mean, nobody in this state can really use the spec yet. Can they? We should have a major bump when something is not backward compatible. So If we change a field from a boolean to a string vs. null, that is still backward compatible. However, if we change an integer to a string, this is not and should therefore have a major version change.

**Comment by eselkin** [comment]
I was just thinking we can have releases that have fixed version increments. Maybe we should keep it to major.minor only without micro. I don't think every pull-request should change the version, but the releases should. We should release versions when milestones are reached (we should add milestones too so we can have a roadmap).
